pkg/scheduler/internal/cache/cache.go

```diff
@@ -734,6 +734,7 @@ func (cache *schedulerCache) cleanupAssumedPods(now time.Time) {
 		ps, ok := cache.podStates[key]
 		if !ok {
 			klog.Fatal("Key found in assumed set but not in podStates. Potentially a logical error.")
+			continue
```

Comment:
calls to klog.Fatal will terminate the program, so this line will never be executed

Reply:
get, I will close this PR. Thanks for review